content/docs/concepts_features/high_availability.md

```diff
@@ -59,3 +59,16 @@ Components `osm-controller` and `osm-injector` allow for separate horizontal sca
 
 
 [1] Headless: usually referred in the control-plane/data-plane design paradigm, refers to the concept that allows, when having a dependency between two components, for the depender agent to survive and keep running with latest state when the dependee dies or becomes unreachable.
+
+### Horizontal Pod Autoscaling - HPA
+HPA will automatically scale up or down control plane pods based on the average target CPU utilization (%). 
+To enable HPA, under the `osm-controller` or `injector` parameters, there is a corresponding `autoScale` -> `enable` boolean field in the [values file](https://github.com/openservicemesh/osm/blob/release-v0.9/charts/osm/values.yaml).
```

Comment:
Also instead, let's suggest to use `osm install --set` or `helm install --set` to update the flags.

Same applies to PDB.